SQL/database_changelog.txt

```diff
@@ -1,3 +1,15 @@
+8th April 2017, by Shadowlight213
+
+Modified table 'player' adding the column 'exp'
+
+ALTER TABLE player ADD exp mediumtext NOT NULL DEFAULT '';
```

Comment:
Ok so, what you want to do here is construct a new table with a structure akin to `ckey|command| security|medical|etc.`. Our reference column is `ckey`, this stores the ckey of the player which this row in the table refers to and will be used to retrieve this row in our query. Following that is each of the department experience columns where you put the actual experience values for the player in as integers. 

After getting most of the way through writing an explanation of an INNER JOIN query I realized you don't actually have to use one because this'd be a client proc and we'll have the player ckey already so you can just do the real simple query: `SELECT command, medical, security, etc. FROM experience WHERE exp.ckey = "[sql_ckey]"`

It helps to picture this as a spreadsheet so:
![capture](https://cloud.githubusercontent.com/assets/4343468/25312355/73d64fbe-285a-11e7-8bdd-db8e01f1881a.PNG)
Note I wrote this on the basis of how the player table will be structured when #26425 is merged as `ckey` will be the primary key of the table then.